superset-frontend/src/components/AsyncAceEditor/AsyncAceEditor.stories.tsx

```diff
@@ -87,11 +87,13 @@ AsyncAceEditor.args = {
 AsyncAceEditor.argTypes = {
   editorType: {
     defaultValue: 'json',
-    control: { type: 'select', options: editorTypes },
+    control: { type: 'select' },
+    options: editorTypes,
   },
   defaultTheme: {
     defaultValue: 'github',
-    control: { type: 'radio', options: ['textmate', 'github'] },
+    control: { type: 'radio' },
+    options: ['textmate', 'github'],
```

Comment:
I can't even tell if this one is working (they look the same?), but it does change whatever content you might've typed into the editor itself. I won't lose too much sleep over that though.

Reply:
You are right! default theme doesn't change anything. But at least the control is there in the storybook if we want to further debug why that is.